packages/mobile/index.js

```diff
@@ -26,4 +27,18 @@ const customErrorHandler = (e, isFatal) => {
 }
 ErrorUtils.setGlobalHandler(customErrorHandler)
 
+// Prevent Text elements font from scaling over 2
```

Comment:
I would double check with @jeanregisser also to make sure this is fine to set globally.

Reply:
@jeanregisser can we set global defaults in the index.js?

Reply:
I think it should be fine, but lets wait for Jean just in case.

Reply:
This could have unintended consequences in 3rd party libs.
Settings this globally should be avoided whenever possible.
Instead we should use a wrapper component with the defaults we want.

Hopefully this won't come bite us in the future ;)